allPostsToWordpress.py

```diff
@@ -21,7 +21,12 @@ def addPost(post, channel, username, wpUrl, doProtected, protectedPassword, doCo
     ET.SubElement(item, 'wp:post_date_gmt').text = pubDate.astimezone(datetime.timezone.utc).strftime('%Y-%m-%d %H:%M:%S')
     ET.SubElement(item, 'wp:comment_status').text = 'open'
     ET.SubElement(item, 'wp:ping_status').text = 'open'
-    ET.SubElement(item, 'wp:post_name').text = post.get('linkId')
+    postTitle = post.find('title').text.strip().lower()
+    wordChar = re.sub('[^\s\w]', '', postTitle)
+    if postTitle == '(no subject)':
+        ET.SubElement(item, 'wp:post_name').text = post.get('linkId')
```

Comment:
This would be a little easier to read if we assigned the name we're going to use to a variable in this if/else (something like `postName`, then afterwards did

    ET.SubElement(item, 'wp:post_name') = postName